.diffplug-pr/pr_2.md

```diff
@@ -0,0 +1,4 @@
+
+### `path/to/thingy`
+
```

Comment:
Multi line comment.

Reply:
I've got something to say about this.

https://github.com/diffplug/T-MATS/blob/74a3671cfcb04bd76f9b113ac98ecc4a14b05970/Trunk/Install_TMATS.m#L26